generators/python/src/fern_python/generators/sdk/client_generator/request_properties.py

```diff
@@ -3,3 +3,36 @@
 
 def request_property_to_name(request_property: ir_types.RequestPropertyValue) -> str:
     return request_property.get_as_union().name.name.snake_case.safe_name
+
+
+def retrieve_pagination_default(type_reference: ir_types.TypeReference) -> int:
+    default_value = None
+    union = type_reference.get_as_union()
+    if union.type == "primitive":
+        maybe_v2_scheme = union.primitive.v_2
+        if maybe_v2_scheme is not None:
+            default_value = maybe_v2_scheme.visit(
+                integer=lambda it: it.default,
+                double=lambda _: None,
+                string=lambda _: None,
+                boolean=lambda _: None,
+                long_=lambda _: None,
+                big_integer=lambda _: None,
+                uint=lambda _: None,
+                uint_64=lambda _: None,
+                date=lambda _: None,
+                date_time=lambda _: None,
+                uuid_=lambda _: None,
+                base_64=lambda _: None,
+                float_=lambda _: None,
```

Comment:
We might need to handle default values typed on other numeric types (e.g. [long](https://github.com/fern-api/fern/blob/6d3eb48a4c090afe7a7627d960027a302c783ca2/packages/ir-sdk/fern/apis/ir-types-latest/definition/types.yml#L258) and [double](https://github.com/fern-api/fern/blob/6d3eb48a4c090afe7a7627d960027a302c783ca2/packages/ir-sdk/fern/apis/ir-types-latest/definition/types.yml#L267) both support default values today). 

For example, if the user defines the parameter in the OpenAPI spec as a `number` instead of an `integer` (re: [docs](https://swagger.io/docs/specification/v3_0/data-models/data-types/#numbers)).

Reply:
Great point. What would be the ideal behavior for long / doubles? Collapsing these values into ints when defaulting the pagination offset?